mappings/com/mojang/blaze3d/texture/TextureUtil.mapping

```diff
@@ -0,0 +1,41 @@
+CLASS com/mojang/blaze3d/platform/TextureUtil com/mojang/blaze3d/texture/TextureUtil
+	FIELD DEFAULT_IMAGE_BUFFER_SIZE DEFAULT_IMAGE_BUFFER_SIZE I
+	FIELD LOGGER LOGGER Lorg/slf4j/Logger;
+	FIELD MIN_MIPMAP_LEVEL MIN_MIPMAP_LEVEL I
+	METHOD bind bind (I)V
+		ARG 0 id
+	METHOD generateTextureId generateTextureId ()I
+	METHOD initTexture initTexture (Ljava/nio/IntBuffer;II)V
+		ARG 0 imageData
+		ARG 1 width
+		ARG 2 height
+	METHOD prepareImage prepareImage (III)V
+		ARG 0 id
+		ARG 1 width
+		ARG 2 height
+	METHOD prepareImage prepareImage (IIII)V
+		ARG 0 id
+		ARG 1 maxLevel
+		ARG 2 width
+		ARG 3 height
+	METHOD prepareImage prepareImage (Lnet/minecraft/unmapped/C_ayikuhxa$C_povfxtzr;III)V
+		ARG 0 internalFormat
+		ARG 1 id
+		ARG 2 width
+		ARG 3 height
+	METHOD prepareImage prepareImage (Lnet/minecraft/unmapped/C_ayikuhxa$C_povfxtzr;IIII)V
+		ARG 0 internalFormat
+		ARG 1 id
+		ARG 2 maxLevel
+		ARG 3 width
+		ARG 4 height
+	METHOD readResource readResource (Ljava/io/InputStream;)Ljava/nio/ByteBuffer;
+		ARG 0 inputStream
+	METHOD releaseTextureId releaseTextureId (I)V
+		ARG 0 id
+	METHOD writeAsPNG writeAsPNG (Ljava/lang/String;IIII)V
```

Comment:
Capitalization, shouldn't this be `writeAsPng` if we're following standard conventions?